Lib/calendar.py

```diff
@@ -261,8 +292,8 @@ def yeardatescalendar(self, year, width=3):
         days. Days are datetime.date objects.
         """
         months = [
```

Comment:
This now fits on one line (part 1)
```suggestion
        months = [self.monthdatescalendar(year, m) for m in Month]
```